stories/Post.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+
+type PostProps = {
+  name: React.ReactNode
+  username: React.ReactNode
+  date: React.ReactNode
+  description: React.ReactNode
+  image: React.ReactNode
+  pfp: React.ReactNode
```

Comment:
Why name is ReactNode? ReactNode is used for div and stuff, not for string argument.
Should be like this (I didn't test this code, should apply anyways...)
```suggestion
  name: string
  username: string
  date: string
  description: string
  image: string
  pfp: string
```